sycl/include/sycl/ext/intel/experimental/esimd/detail/simd_view_impl.hpp

```diff
@@ -336,6 +336,15 @@ template <typename BaseTy, typename RegionTy> class simd_view_impl {
     return v[i];
   }
 
+  /// Read a single element from a 1D region, by value only.
+  template <typename T = simd_view_impl,
+            typename = sycl::detail::enable_if_t<T::is1D()>>
+  __SYCL_DEPRECATED("use operator[] form.")
+  element_type operator()(int i) const {
```

Comment:
This only supports returning a scalar value with () operator. Is there a change to add support for returning a writable view from simd_view as requested?

Reply:
@kychendev, yes, I'm working on that.

Reply:
@kychendev, it turned out to be not that easy to do. I prefer to do it in further patches. But I'm working on it in the meantime.